static/src/javascripts/projects/common/modules/save-for-later.js

```diff
@@ -51,6 +51,15 @@ define([
         this.isContent = !/Network Front|Section/.test(config.page.contentType);
         this.userData = null;
         this.savedArticlesUrl = config.page.idUrl + '/saved-for-later-page';
+
+        _.bindAll(this,
+            'save',
+            'delete',
+            'onSaveArticle',
+            'onDeleteArticle',
+            'createSaveFaciaItemHandler',
+            'createDeleteFaciaItemHandler'
+        );
```

Comment:
I wonder if you could do `_.bindAll(this, Object.keys(SaveForLater.prototype))`.